nodes/iol/iol.go

```diff
@@ -95,7 +95,8 @@ func (n *iol) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 
 	n.Cfg.Binds = append(n.Cfg.Binds,
 		// mount nvram so that config persists
-		fmt.Sprint(path.Join(n.Cfg.LabDir, "nvram"), ":/iol/nvram_00001"),
+		fmt.Sprint(path.Join(n.Cfg.LabDir, "nvram_00001"), ":/iol/nvram_00001"),
+		fmt.Sprint(path.Join(n.Cfg.LabDir, "nvram_00003"), ":/iol/nvram_00003"),
```

Comment:
@kaelemc would you mind introducing constants here for nvram1 and nvram2?

Reply:
Fixed.